src/lib/core/components/visualizations/implementations/MosaicVisualization.tsx

```diff
@@ -530,6 +530,10 @@ function MosaicViz(props: Props<Options>) {
                   facetVariable ? facetVariable.displayName : 'Facet'
                 }
                 enableSpinner={data.pending}
+                selectedReferenceValues={[
+                  xAxisReferenceValue,
+                  yAxisReferenceValue,
+                ]}
```

Comment:
This could use a `useMemo` to prevent extra work in the `ContingencyTable` component:

```typescript
const selectedReferenceValues = useMemo(() => [
  xAxisReferenceValue,
  yAxisReferenceValue
], [xAxisReferenceValue, yAxisReferenceValue]);
```